### DISCUSSION TITLE
Remove 12 Hours of Data After Ubuntu Hung

### DISCUSSION THREAD

**Discussion by lukedbarton** [jasonacox/Powerwall-Dashboard#92] [Q&A] [answered]
Hey All,
Loving the Dashboard!
Just wondering if anyone could provide any advice of how I could remove some data from influx? 
My instance of this great tools looked to loose connectivity with my powerwall and has flat lined usage as can be seen in the screenshot.
Does anyone know if there is a way I can remove this subset of data so it simple states 0?

![image](https://user-images.githubusercontent.com/69992302/193478868-c921d75c-8338-4f79-b79d-cb6eb9eba17e.png)

Cheers
Luke

**Comment by mcbirse** [comment, marked as answer]
Hi @lukedbarton 

There actually won't be any power data for you to remove, as it wasn't recorded. The graph will just straight line between data points, which is why it looks like that if you're missing a large chunk of data. It is actually useful to identify drop outs like this.

This is a perfect use case for a tool I am working on, so don't worry about it for now. 😉 

I'm working on a tool we can use to import missing data from the Tesla cloud (i.e. Tesla app data). It won't be as accurate as the dashboard (per 5 min data only), but will be good to fix issues like this (some data is better than no data) and also you could import historical data from before you installed the dashboard, if desired.

**Reply by lukedbarton** [reply]
Thats awesome thank you very much for the explanation and the tool sounds amazing. How good is data :-)
Cheers again.
